g3doc/tutorials/graph_keras_cnn_flowers.ipynb

```diff
@@ -0,0 +1,1198 @@
+      "metadata": {
+        "id": "24gYiJcWNlpA"
+      },
+      "source": [
+        "##### Copyright 2021 Google LLC"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ioaprt5q5US7",
+        "cellView": "form"
+      },
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ItXfxkxvosLH"
+      },
+      "source": [
+        "# Graph regularization for image classification using synthesized graphs\n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/neural_structured_learning/tutorials/graph_keras_cnn_flowers\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/neural-structured-learning/blob/master/g3doc/tutorials/graph_keras_cnn_flowers.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/neural-structured-learning/blob/master/g3doc/tutorials/graph_keras_cnn_flowers.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "z3otbdCMmJiJ"
+      },
+      "source": [
+        "## Overview"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Eg62Pmz3o83v"
+      },
+      "source": [
+        "This notebook is a sister notebook to this example - [Graph regularization for sentiment \n",
+        "classification using synthesized graphs](https://www.tensorflow.org/neural_structured_learning/tutorials/graph_keras_lstm_imdb). In here, we'll build a flower classification model that can\n",
+        "categorize images of flowers into discrete classes such as sunflowers, roses,\n",
+        "tulips, etc. \n",
```

Comment:
Reword this sentence as "This notebook is the counterpart of [Graph regularization for sentiment classification using synthesized graphs](https://www.tensorflow.org/neural_structured_learning/tutorials/graph_keras_lstm_imdb) for image classification. In this notebook, we will build a flower classification model to categorize images of flowers into discrete classes such as sunflowers, roses, tulips, and so on."